patterns/core/sass/settings/_colors.scss

```diff
@@ -0,0 +1,7 @@
+$ColorScheme: (
```

Comment:
It'll be going in the Contributing guidelines #66 

This starts to get in to documenting our settings. We don't need to document `ColorScheme`, but we will need to document the different settings (we're using [Toolkit's Settings](https://github.com/at-import/toolkit#settings))